fastlane/Fastfile

```diff
@@ -9,11 +9,16 @@
 default_platform(:android)
 
 platform :android do
-  desc "Runs all the tests"
+  desc "Runs all unit tests"
   lane :test do
     gradle(task: "test")
   end
 
+  desc "Runs all UI tests"
```

Comment:
As we pass the script to the github action, this is currently not used, can we remove it? I tried to use but the bundle wasn't available in the action's environemnt

Reply:
Agree; I will remove it 👍